datajoint/external.py

```diff
@@ -73,6 +73,12 @@ def s3(self):
             self._s3 = s3.Folder(**self.spec)
         return self._s3
 
+    @property
+    def gcs(self):
+        # TODO: Add gcsfs dependency?
```

Comment:
Yes, adding gcsfs dependency would be consistent with how S3 was managed.

Reply:
OK.  Add dependency.